common/scripted_effects/wc_sacrifice_scripted_effects.txt

```diff
@@ -138,18 +138,115 @@ caltulate_PREV_offmap_old_gods_currency_insanity_effect = {
 				}
 				else_if = {
 					limit = { tier = KING }
-					PREV = { change_variable = { which = local_received_offmap_old_gods_currency_insanity_variable value = 300 } }
+					PREV = { change_variable = { which = local_received_offmap_old_gods_currency_insanity_variable value = 350 } }
 				}
 				else = {
 					limit = { tier = EMPEROR }
-					PREV = { change_variable = { which = local_received_offmap_old_gods_currency_insanity_variable value = 500 } }
+					PREV = { change_variable = { which = local_received_offmap_old_gods_currency_insanity_variable value = 650 } }
 				}
 			}
-			# Child
+			#Class
```

Comment:
Too long, add tooltips.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/78401364-a5671180-7609-11ea-8268-2c3d5bf4ab85.png)

</details>

Reply:
its now way shorter

Reply:
> its now way shorter

Strong words. I still see the list of all the magical classes that covers 40% of my screen.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/78459006-f983fb80-76c6-11ea-8ac8-51042a9dfad8.png)

</details>

Reply:
so do you want me to change the trigger to include tooltip?

Reply:
> so do you want me to change the trigger to include tooltip?

When did you learn to read thoughts?